### PR TITLE
Updated water storage and curly leaves because Bence asked us to.

### PR DESCRIPTION
**Pull request by Limhes**

### CODE REVIEW

File: sequioa/crown.md [hunk 1]
```diff
@@ -10,3 +10,4 @@ Provide the support and connection between the higher level sub-modules and the
 
 Display green colour.
 Handle chemical data conversion and output.
+Has a nice curly shape for extra pleasure.
```
Comment (thms317) [inline sequioa/crown.md:13]: This comment is sooooo inappropriate, I feel offended

File: sequioa/trunk.md [hunk 2]
```diff
@@ -14,3 +14,8 @@ Greatly improve resistance to overheating in fire situation.
 ### Redcuded sap content
 
 Keeping the `bark` dry means we leave less opporunities for things to catch on fire.
+
+
+## Water storage
+
+In case the tree gets thirsty. Buckets full of juicy water.
```
Comment (thms317) [inline sequioa/trunk.md:21]: Please rename to juice-bucket

**Review by thms317 — CHANGES_REQUESTED** [review]
Unhappy, please change everything.

File: sequioa/crown.md (hunk 1 above)
Comment (Limhes) [reply sequioa/crown.md:13]: Your offence makes me sad. Crying silently

**Comment by bkarpati** [comment]
Thanks for participating in the Git workshop. I'm going to close this PR for now and also delete the remote branch.
Please stay tuned for the follow-up.

**Pull request closed without merging**